File: src/main/java/com/wxl/dyttcrawler/scheduler/local/BatchHashSetDuplicateRemover.java
package com.wxl.dyttcrawler.scheduler.local;

import com.wxl.dyttcrawler.scheduler.BatchDuplicateRemover;
import us.codecraft.webmagic.scheduler.component.HashSetDuplicateRemover;

/**
 * Create by wuxingle on 2020/7/20
 * hashSet保存访问过的request
 */
public class BatchHashSetDuplicateRemover extends HashSetDuplicateRemover
        implements BatchDuplicateRemover {

}

File: src/main/kotlin/com/wxl/dyttcrawler/scheduler/local/BatchHashSetDuplicateRemover.kt
package com.wxl.dyttcrawler.scheduler.local

import com.wxl.dyttcrawler.scheduler.BatchDuplicateRemover
import us.codecraft.webmagic.scheduler.component.HashSetDuplicateRemover

/**
 * Create by wuxingle on 2021/10/11
 * hashSet保存访问过的request
 */
class BatchHashSetDuplicateRemover : HashSetDuplicateRemover(), BatchDuplicateRemover {


}

